src/near-rpc-provider.ts

```diff
@@ -312,4 +295,27 @@ export class NearRpcProvider extends JsonRpcProvider {
       })
     }
   }
+
+  async getBlockWithChunk(params: Record<string, any>): Promise<GetBlockWithChunkRpcResponse> {
```

Comment:
I think this should return `Promise<NearBlockWithChunk>`

Reply:
Done!